src/docs/contributing/linking.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Linking Variables
+noindex: true
+sidebar_order: 4
+---
+
+We create a variety of links in our documentation, but the most commonly-used link type connects content either between product guides and the platform pages or between platform pages themselves. Each of these may take a different format, depending on whether a platform needs to be specified.
+
```

Comment:
Adding an H2 or the RH sidebar won't render.

```suggestion

## Product & Platform Links

```

Reply:
oof. Thanks.